src/game.cpp

```diff
@@ -8668,7 +8668,8 @@ point game::look_around(WINDOW *w_info, const point pairCoordsFirst)
     const int offset_x = (u.posx() + u.view_offset_x) - getmaxx(w_terrain) / 2;
     const int offset_y = (u.posy() + u.view_offset_y) - getmaxy(w_terrain) / 2;
 
-    int lx = u.posx() + u.view_offset_x, ly = u.posy() + u.view_offset_y;
+    int lx = u.posx() + u.view_offset_x, ly = u.posy() + u.view_offset_y, lz = u.posz() + 0;
+    tripoint lp( lx, ly, lz );
```

Comment:
You could avoid the overriding of `lp` below by using this neat reference trick:
```C++
tripoint lp( u.posx() + u.view_offset_x, ... );
int &lx = lp.x;
int &ly = lp.y;
int &lz = lp.z;
```
If you write to `lx`, it will automatically update `lp` because `lx` and `lp.x` are the same place in memory.